demo/__init__.py

```diff
@@ -12,6 +12,11 @@
 
 from six.moves import range
 
+
+local = 'localhost'
+remote = 'repo.uptane.org'
```

Comment:
A comment saying so might be helpful.  Someone else in the future might attempt to delete this variable after noticing it goes unused.